### PR TITLE
added crypto_kx_* functions

### PR DESCRIPTION
**Pull request by bblaz**
I have added the basic key exchange functions.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -11,7 +11,7 @@ def read(fname):
 
 setup(
     name="pysodium",
-    version="0.6.12",
+    version="0.6.13dev0",
```
Comment (stef) [inline setup.py]: you can make this 13 without the dev0

File: test/test_pysodium.py [hunk 2]
```diff
@@ -303,5 +303,15 @@ def test_crypto_auth(self):
         tag = pysodium.crypto_auth("howdy", sk)
         pysodium.crypto_auth_verify(tag, "howdy", sk)
 
+    def test_crypto_kx(self):
```
Comment (stef) [inline test/test_pysodium.py]: this should have a sodium version check, which also fixes travis failing

**Comment by bblaz** [comment]
Thanks for the comments. Should all be fixed now.

**Comment by stef** [comment]
thanks for all! & welcome to the authors file :)

**Pull request merged**